src/services/StructuralMetadataUtils.js

```diff
@@ -78,25 +78,24 @@ export default class StructuralMetadataUtils {
 
     // Convert time to HH:mm:ss.ms format to use in validation logic
     let convertToSeconds = time => {
```

Comment:
It looks like the approach taken here is converging on what we have in our ruby code:
https://github.com/avalonmediasystem/avalon/blob/develop/app/helpers/application_helper.rb#L173-L182
Or here:
https://github.com/avalonmediasystem/avalon/blob/develop/app/controllers/application_controller.rb#L129-L147